webtranslate/project_type.py

```diff
@@ -181,168 +198,159 @@ def get_translated_cmd(self):
-    ParameterInfo("STATION",        [P__],      None, False, True ),
+    ParameterInfo("NBSP", [], None, False, False),
+    ParameterInfo("COPYRIGHT", [], None, False, True),
+    ParameterInfo("TRAIN", [], None, False, True),
+    ParameterInfo("LORRY", [], None, False, True),
+    ParameterInfo("BUS", [], None, False, True),
+    ParameterInfo("PLANE", [], None, False, True),
+    ParameterInfo("SHIP", [], None, False, True),
+    ParameterInfo("TINYFONT", [], None, False, True),
+    ParameterInfo("BIGFONT", [], None, False, True),
+    ParameterInfo("BLUE", [], None, False, False),
+    ParameterInfo("SILVER", [], None, False, False),
+    ParameterInfo("GOLD", [], None, False, False),
+    ParameterInfo("RED", [], None, False, False),
+    ParameterInfo("PURPLE", [], None, False, False),
+    ParameterInfo("LTBROWN", [], None, False, False),
+    ParameterInfo("ORANGE", [], None, False, False),
+    ParameterInfo("GREEN", [], None, False, False),
+    ParameterInfo("YELLOW", [], None, False, False),
+    ParameterInfo("DKGREEN", [], None, False, False),
+    ParameterInfo("CREAM", [], None, False, False),
+    ParameterInfo("BROWN", [], None, False, False),
+    ParameterInfo("WHITE", [], None, False, False),
+    ParameterInfo("LTBLUE", [], None, False, False),
+    ParameterInfo("GRAY", [], None, False, False),
+    ParameterInfo("DKBLUE", [], None, False, False),
+    ParameterInfo("BLACK", [], None, False, False),
+    ParameterInfo("PUSH_COLOUR", [], None, False, False),
+    ParameterInfo("POP_COLOUR", [], None, False, False),
+    ParameterInfo("COMMA", [PP_], 0, False, True),
+    ParameterInfo("SIGNED_WORD", [PP_], 0, False, True),
+    ParameterInfo("UNSIGNED_WORD", [PP_], 0, False, True),
+    ParameterInfo("CURRENCY", [PP_], 0, False, True),
+    ParameterInfo("VELOCITY", [PP_], 0, False, True),
+    ParameterInfo("VOLUME", [PP_], 0, False, True),
+    ParameterInfo("VOLUME_SHORT", [PP_], 0, False, True),
+    ParameterInfo("POWER", [PP_], 0, False, True),
+    ParameterInfo("WEIGHT", [PP_], 0, False, True),
+    ParameterInfo("WEIGHT_SHORT", [PP_], 0, False, True),
+    ParameterInfo("CARGO_LONG", [P_G, PP_], 1, False, True),
+    ParameterInfo("CARGO_SHORT", [P__, PP_], 1, False, True),  # short cargo description, only ### tons, or ### litres
+    ParameterInfo("CARGO_TINY", [P__, PP_], 1, False, True),  # tiny cargo description with only the amount
+    ParameterInfo("CARGO_NAME", [P_G], None, True, True),
+    ParameterInfo("HEX", [PP_], 0, False, True),
+    ParameterInfo("STRING", [P_G], None, True, True),
+    ParameterInfo("DATE1920_LONG", [P__], None, False, True),
+    ParameterInfo("DATE1920_SHORT", [P__], None, False, True),
+    ParameterInfo("DATE_LONG", [P__], None, False, True),
+    ParameterInfo("DATE_SHORT", [P__], None, False, True),
+    ParameterInfo("POP_WORD", [P__], None, False, True),
+    ParameterInfo("STATION", [P__], None, False, True),
```

Comment:
some more fmt: off blocks in this file, i think

Reply:
Same as the other one .. any change to this list is such a nag if you want to keep it aligned with spaces.

Reply:
I don't care about the short tables. But the tables in project_type.py are impossible to read without alignment.
Their content is mostly arbitrary (because newgrf specs), so readability is the only thing that helps maintaining them.
In the unaligned auto-formatted case, you could as well replace them with a pickled dump :) It's like nforenum.